examples/simple/webpack.config.js

```diff
@@ -32,55 +32,55 @@ module.exports = {
                 '..',
                 'packages',
                 'ra-core',
-                'src'
+                'lib'
```

Comment:
Are you sure about this ? It looks like it's going to take the `cjs` files, thus not building them itself when changed

Reply:
Not sure this is the best solution, but it works on my machine®!
I considered your solution but is it the role of the simple example to build the RA sources? Idk

On the CI, the build is done before the test, hence it avoid to double-build these sources.

Reply:
Not its role but invaluable when working on one of those packages

Reply:
Then, just to be sure, you suggest that I should add all the babel plugins related to the all the packages in `examples/simple/.babelrc`?
Or I would just use the same and find somewhere to add `BABEL_ENV=cjs`.

Reply:
I don't what would work best tbh. IMO, adding the `BABEL_ENV=cjs` looks easier

Reply:
Don't forget our windows users though ;)

Reply:
Don't change the simple example webpack config. We use a direct link to src because it lets us develop on the packages and see the impact in the example *without rebuilding*. Your change would break that.